nixos/modules/services/x11/window-managers/exwm.nix

```diff
@@ -38,6 +38,13 @@ in
           file.
         '';
       };
+      package = mkOption {
+        type = types.package;
+        default = pkgs.emacs;
+        defaultText = literalExpression "pkgs.emacs";
+        description = "Which Emacs package to use for EXWM.";
+        example = literalExpression "pkgs.emacs-gtk";
+      };
```

Comment:
This can be simplified by using [`lib.mkPackageOption`](https://noogle.dev/f/lib/mkPackageOption).

Reply:
Thank you for telling me about that! I've changed it.